frontend/src/components/gallerys/StorageGallery.tsx

```diff
@@ -8,9 +8,9 @@ type Props = {
 
 function StorageGallery(props: Props) {
     return (
-        <div>
+        <>
             {props.storages.map(current => <StorageCard storage={current}/>)}
```

Comment:
Würde es dennoch in ein <div> setzen, das ist aber nur ein schönheits ding zur Zeit